include/system_headers/multiboot.h

```diff
@@ -269,6 +269,13 @@ struct multiboot_apm_info
   multiboot_uint16_t dseg_len;
 };
 
+typedef struct
+{
+  multiboot_uint32_t RSDP;
+  // EFI_RUNTIME_SERVICES *RT;
+  void *RT;
```

Comment:
This struct cannot go here, because the multiboot header is not Tilck's. It's 3rd-party code ("system_headers"). We do not touch such headers, unless it's impossible to make the project compile.

This struct should go in a dedicated in the common area between the kernel, the bootloader and everything else. So, make it be: `include/tilck/common/boot.h`.

Also, if you notice, there are no "void *" pointers in the multiboot header for a precise reason: the size has to be the same no matter what (the bitness of the bootloader vs the kernel might differ). Also, the data might not be accessible directly as in userspace. When dealing with the barrier between the bootloader and the kernel, addresses are typically integers as they cannot be simply de-referenced, unless there's a mapping in kernel's virtual space or paging is entirely disabled.

Therefore, just use `uint32` for both the values. Note that when Tilck is ported to x86-64, this will remain and continue to work: if you check the call to AllocatePages(), we're passing a MAX_ADDRESS which is  very low (EFI_MBI_MAX_ADDR = 64 KB). 

Finally, as per Tilck's coding style, no typedef.